examples/fscloud/main.tf

```diff
@@ -86,23 +93,19 @@ module "cbr_account_level" {
 
   # Demonstrates how additional context to the rules created by this module can be added.
   # This example open up:
-  #   1. Flows from icd mongodb, postgresql to kms on private endpoint
-  #   2. Flow from schematics on public kms endpoint
-  #   3. Add a block of ips to schematics public endpoint
-  #   4. Flow from vpc(s) specified in input zone_vpc_crn_list to postgresql private endpoint
-  custom_rule_contexts_by_service = {
-    "kms" = [{
-      endpointType      = "private"
-      service_ref_names = ["databases-for-mongodb", "databases-for-postgresql"]
-      },
-      {
-        endpointType      = "public"
-        service_ref_names = ["schematics"]
+  #   1. Flow from schematics on public HPCS endpoint
+  #   2. Add a block of ips to schematics public endpoint
+  #   3. Flow from vpc(s) specified in input zone_vpc_crn_list to postgresql private endpoint
+  #   4. Flows from RabbitMQ to KMS on private endpoint
```

Comment:
I believe you meant to comment about Postgresql and not rabbitMQ? There is no trace of RabbitMQ in the code.

Reply:
I agree Vincent, I have made changes to the comments accordingly.